### ISSUE TITLE
Oxlint is marked with the error color in `WSL` as the color of Warn

### ISSUE THREAD

**Issue by mozhuanzuojing** [oxc-project/oxc#6489]
![image](https://github.com/user-attachments/assets/f6501fb9-6d81-483d-a818-1127d08031a4)

> This problem has been around for a long time

#### WSL2 env：
Version: 1.94.2 (system setup)
Commit: 384ff7382de624fb94dbaf6da11977bba1ecd427
Date: 2024-10-09T16:08:44.566Z
Electron: 30.5.1
ElectronBuildId: 10262041
Chromium: 124.0.6367.243
Node.js: 20.16.0
V8: 12.4.254.20-electron.0
OS: Windows_NT x64 10.0.19045

**Comment by pumano** [comment]
@mozhuanzuojing  Looks like eslint config not applied, because default oxlint is warn

**Comment by mozhuanzuojing** [comment]
> @mozhuanzuojing Looks like eslint config not applied, because default oxlint is warn

That being said, oxlint doesn't read the config file `.eslintrc` correctly under `WSL`

**Comment by mozhuanzuojing** [comment]
![image](https://github.com/user-attachments/assets/886d0b38-e27d-4637-b639-6e60f8acce87)

**Comment by DonIsaac** [comment]
It looks like your `.eslintrc` is not in your root directory. You should either move it to your project's root, or configure the extension to look for a config file at `page/.eslintrc`.

**Comment by mozhuanzuojing** [comment]
> It looks like your `.eslintrc` is not in your root directory. You should either move it to your project's root, or configure the extension to look for a config file at `page/.eslintrc`.

I see, the oxlint plugin doesn't support `.eslintrc` of front-end projects in `vscode workspace`

**Comment by DonIsaac** [comment]
It does, just not by default. Again, you'll need to edit the extension settings.

**Comment by mozhuanzuojing** [comment]
> It does, just not by default. Again, you'll need to edit the extension settings.

![image](https://github.com/user-attachments/assets/3446d282-0052-421b-b99d-ddc234d06791)

Even if you configure "page/.eslintrc", it does not take effect.

**Comment by DonIsaac** [comment]
Hmmm ok, I'll take a look in to that

**Comment by IWANABETHATGUY** [comment]
Can you run `oxlint` and send the result?

**Comment by mozhuanzuojing** [comment]
> Can you run `oxlint` and send the result?

![image](https://github.com/user-attachments/assets/48d08c63-4f43-4c92-b028-3d4f27243919)

**Comment by IWANABETHATGUY** [comment]
Yeah, you could see they are all warnings, also language server just uses `page/.eslintrc` as configuration file

**Comment by IWANABETHATGUY** [comment]
Looks like related to workspace

**Comment by pumano** [comment]
0.10.2 - fixed and works correctly
<img width="746" alt="Screenshot 2024-10-23 at 00 51 23" src="https://github.com/user-attachments/assets/114303cd-f7ce-4671-ba81-eefb0d76e12d">

**Comment by mozhuanzuojing** [comment]
> Version 0.10.2 has a small problem
![image](https://github.com/user-attachments/assets/686d0bac-d1b1-465a-aef2-d47c14b8817d)

**Comment by mozhuanzuojing** [comment]
> 0.10.2 - fixed and works correctly <img alt="Screenshot 2024-10-23 at 00 51 23" width="746" src="https://private-user-images.githubusercontent.com/17675259/379031148-114303cd-f7ce-4671-ba81-eefb0d76e12d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjk2NDU0MTUsIm5iZiI6MTcyOTY0NTExNSwicGF0aCI6Ii8xNzY3NTI1OS8zNzkwMzExNDgtMTE0MzAzY2QtZjdjZS00NjcxLWJhODEtZWVmYjBkNzZlMTJkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMjMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDIzVDAwNTgzNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWI3NWVmNzJhZTI3MTNkYTNjY2Q2YzA4YjUxYmMzYTVhNjBiODZlNmZmNzlmMjA3MTkyZTA0YmI5OTgwZTJhY2ImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.fHoLUEpRWjEq9EstyDIp3uQBRrbDkX-stZv3y65LSdg">
> 
> @DonIsaac cc

no, still not in effect

![image](https://github.com/user-attachments/assets/9532161e-1945-4c3d-bbe3-23d45a1bad32)

**Comment by Boshen** [comment]
Close as "require proper reproduction".
